### PR TITLE
WIP: Docker

### LINKED ISSUE #1: Use Docker to set up environment for build
Travis-CI can also use Docker: https://docs.travis-ci.com/user/docker/.

Example: https://github.com/jfrazelle/dockerfiles

### PR DESCRIPTION
**Pull request by StayerX**
This will close #1  #34 

The Docker file was moved to the root of the project in order to have global access to the files such as the travis-ci files.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# Define environment variable
+# ENV NAME World
+
+RUN apt-get update -y
+RUN apt-get install -y sudo
+RUN apt-get install -y git
+RUN git clone https://github.com/project-renard/devops.git ~/project-renard/devops
+RUN git clone https://github.com/project-renard/curie.git  ~/project-renard/curie
+RUN git clone https://github.com/project-renard/test-data.git  ~/project-renard/test-data
```
Comment (StayerX) [inline Dockerfile:27]: remember to use the cloned repository.

**Comment by zmughal** [comment]
In order to use Xming (for Windows) with Docker, you need to start Xming using XLaunch and turn off the access control:
xlaunch_2017-05-21_19-21-37

Then, you need to set the `$DISPLAY` to the IP address of the host:

```shell
docker run -ti -e DISPLAY="192.168.33.1:0.0"  projectrenard/curie bash -c "perl ~/project-renard/curie/bin/curie"
```

**Comment by zmughal** [comment]
In order to use VNC with Docker, we need `apt-get install vnc4server`, `Xvnc <display-id>` and do a port mapping to the VNC port.

**Comment by StayerX** [comment]
I have added the VNC and the Xming to the README.md.
This also works from git bash :  `export DISPLAY="192.168.99.1:0.0"`

**Comment by zmughal** [comment]
Closing in favour of <https://github.com/oberth-manoeuvre/oberth-prototype>.

**Pull request closed without merging**